lib/ips/summary_operation.rb

```diff
@@ -34,20 +34,26 @@ class SummaryOperation < Inferno::TestGroup
     end
 
     test do
-      title 'IPS Server returns Bundle resource for Patient/id/$summary operation'
+      title 'IPS Server returns Bundle resource for Patient/[id]/$summary GET operation'
       description %(
-        IPS Server return valid IPS Bundle resource as successful result of $summary operation.
+        IPS Server returns a valid IPS Bundle resource as successful result of
+        $summary operation.
 
-        This is required to be a POST per the [guidance](http://hl7.org/fhir/uv/ips/STU1.1/ipsGeneration.html):
-
-        >  This operation returns an IPS document Bundle in response to a POST request.
+        This test currently only issues a GET request for the summary due to a
+        limitation in Inferno in issuing POST requests that omit a Content-Type
+        header when the body is empty. Inferno currently adds a `Content-Type:
+        application/x-www-form-urlencoded` header when issuing a POST with no
+        body, which causes issues in known reference implementations.
 
+        A future update to this test suite should include a required POST
+        request as well as an optional GET request for this content.
       )
+
       input :patient_id
       makes_request :summary_operation
 
       run do
-        fhir_operation("Patient/#{patient_id}/$summary", name: :summary_operation, operation_method: :post)
+        fhir_operation("Patient/#{patient_id}/$summary", name: :summary_operation, operation_method: :get)
```

Comment:
Not resolvable in this file and also not a big deal, but the request URL is `https://hl7-ips-server.hl7.org/fhir//Patient/15/$summary` - we could remove the double forward slash

Reply:
I noticed that but was a little lazy about addressing it because it didn't seem critical at the time.  Good catch thogh. I updated the preset to not have a trailing slash which fixes it for the preset case.  However, `fhir_operation` should be a little more defensive in the URLs it builds to query if possible.  I'll open a ticket to fix the general case in inferno-core.